Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -26,11 +26,11 @@
         public mainGrid: IKendoGrid<IItSystemUsageOverview>;
         public mainGridOptions: kendo.ui.GridOptions;
 
-        public usageGrid: kendo.ui.Grid;
-        public modal: kendo.ui.Window;
+        //public usageGrid: kendo.ui.Grid;
```

Comment:
I commented these because they've removed from the interface. But not from the class. So I removed them from the class.

It's not deleted because it's coming back as some point :)